pyproject.toml

```diff
@@ -10,18 +10,19 @@ requires-python = "<3.12,>=3.11"
 dependencies = [
     "babel==2.15.0",
     "mex-common@git+https://github.com/robert-koch-institut/mex-common.git@0.32.0",
+    "pyyaml==6.0.1",
     "reflex==0.5.7",
     "uvicorn==0.20.0",
 ]
 optional-dependencies.dev = [
     "ipdb==0.13.13",
     "mypy==1.11.0",
-    "pytest-cov==5.0.0",
```

Comment:
sadly not that easily because the editor runs in a different process. but i'm also not sure how helpful it would be for this sort of visual / browser-based testing